Makefile

```diff
@@ -13,3 +18,7 @@ upgrade:
 		--workdir /usr/src/app \
 		--volume $(PWD):/usr/src/app \
 		$(IMAGE_NAME) ./bin/upgrade.sh
+
+release:
+	docker tag $(IMAGE_NAME) $(RELEASE_REGISTRY)/codeclimate-phpmd:$(RELEASE_TAG)
```

Comment:
Shouldn't this be `codeclimate-pmd`? Same for line below.

Reply:
whoops, yeah, got confused with the other plugin, nice catch!